recipients_app/lib/data/repositories/messaging_repository.dart

```diff
@@ -1,12 +1,17 @@
-import "dart:developer";
+// ignore_for_file: unreachable_from_main
 
+import "dart:developer";
 import "package:firebase_messaging/firebase_messaging.dart";
 
 /// This must be a top level function in order to work
-Future<void> handleBackgroundMessage(RemoteMessage message) async {
-  log("PUSHNOTIFICATION title: ${message.notification?.title}");
-  log("PUSHNOTIFICATION body: ${message.notification?.body}");
-  log("PUSHNOTIFICATION data: ${message.data}");
+@pragma("vm:entry-point")
```

Comment:
When using Flutter version 3.3.0 or higher, the message handler must be annotated with @pragma('vm:entry-point') right above the function declaration (otherwise it may be removed during tree shaking for release mode).